### PR TITLE
{math}[gompi/2021a] Mmg v5.6.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/Mmg/Mmg-5.6.0-gompi-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+configopts = '-DUSE_SCOTCH=ON '
+configopts += '-DLIBMMG2D_SHARED=ON -DLIBMMG3D_SHARED=ON -DLIBMMGS_SHARED=ON -DLIBMMG_SHARED=ON '
+
+sanity_check_paths = {
+    'files': ['bin/%s_O3' % x for x in ['mmg2d', 'mmg3d', 'mmgs']] +
+             ['lib/lib%s.%s' % (x, y) for x in ['mmg2d', 'mmg3d', 'mmgs', 'mmg'] for y in ['a', SHLIB_EXT]],
+    'dirs': ['include/mmg/%s' % x for x in ['mmg2d', 'mmg3d', 'mmgs']]
+}
+
```
Comment (casparvl) [inline easybuild/easyconfigs/m/Mmg/Mmg-5.6.0-gompi-2021a.eb:39]: Is there a sensiblte `santicy_check_commands`  that you could set (i.e. a `--version` or something) to check the binaries will start?

**Review by casparvl — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/Mmg/Mmg-5.6.0-gompi-2021a.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/m/Mmg/Mmg-5.6.0-gompi-2021a.eb:39]: I got curious and worked on a variant for 2021b, #15622
We can (and should) add a test step, but sanity checks won't work (bad return values)

File: easybuild/easyconfigs/m/Mmg/Mmg-5.6.0-gompi-2021a.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+toolchain = {'name': 'gompi', 'version': '2021a'}
+
+source_urls = ['https://github.com/MmgTools/mmg/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['bbf9163d65bc6e0f81dd3acc5a51e4a8c47a7fdae849abc26277e01154fe2437']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('binutils', '2.36.1'),
```
[the remark this answers is not in the document]
Comment (sassy-crick) [reply easybuild/easyconfigs/m/Mmg/Mmg-5.6.0-gompi-2021a.eb:30]: You need `CMake`, but you don't need `binutils` as that is loaded as a requirement. Thus I removed `binutils`.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Review by lexming — APPROVED** [review]
@sassy-crick Thanks for this contribution. I just updated the PR to current `develop` and made both easyconfigs of Mmg v5.6.0 to be equivalent.

LGTM

**Comment by lexming** [comment]
Going in, thanks @sassy-crick

**Pull request merged**